### ISSUE TITLE
Split line

### ISSUE THREAD

**Issue by skerit** [banga/powerline-shell#154]
I would like the prompt to start on a new line.
I added a regular newline character, but then I get this result:

![powerline](https://cloud.githubusercontent.com/assets/755212/3561295/e329add2-09b5-11e4-89cd-30c67e0210d9.png)

It's a small beauty problem, but it really annoys me.

**Comment by t3chguy** [comment]
Hello, sorry for re-awaking this Issue;
but I am wanting to achieve what you have @skerit 
I can't seem to be able to figure out how to drop the "$" onto the second line, currently only the actual prompt is on the second line for me :/

**Comment by manasb** [comment]
I wanted to do the same thing. Thanks for the help.

**Comment by b-ryan** [comment]
I'm in the same boat as all of you. I haven't yet figured out how to get this to behave properly.

**Comment by manasb** [comment]
b-ryan, if you would like to just have the prompt dropped down to the next line, find ```def draw(self)``` in ```powerline-shell.py``` and at the end of the line, add in a ```+ '\n\r' ``` but don't run install.py again. This is what I found to work.

For dropping the '$' down to the next line, I'm not entirely sure but you might have to add in a newline character somewhere. skerit probably would be able to tell you where the edit needs to be made.

**Comment by szepeviktor** [comment]
Is my comment above a hack?

**Comment by BrianEnigma** [comment]
I've had success with:

```python
diff --git a/powerline_shell_base.py b/powerline_shell_base.py
index 8b95fb3..05f3d72 100755
--- a/powerline_shell_base.py
+++ b/powerline_shell_base.py
@@ -56,6 +57,8 @@ class Powerline:
     def color(self, prefix, code):
         if code is None:
             return ''
+        elif code is -1:
+            return self.color_template % ('[0m')
         else:
             return self.color_template % ('[%s;5;%sm' % (prefix, code))

diff --git a/segments/root.py b/segments/root.py
index 9cd5e8c..9982d2b 100644
--- a/segments/root.py
+++ b/segments/root.py
@@ -9,6 +9,7 @@ def add_root_indicator_segment():
     if powerline.args.prev_error != 0:
         fg = Color.CMD_FAILED_FG
         bg = Color.CMD_FAILED_BG
+    powerline.append("\n", -1, -1, '')
     powerline.append(root_indicators[powerline.args.shell], fg, bg)

 add_root_indicator_segment()
```

If I were to make a PR with this change, I'd make the newline conditional upon an environment variable or configuration option.

**Comment by b-ryan** [comment]
@szepeviktor I didn't even notice your comment. I had been adding `\n` without the carriage return. But in terms of making this work for everyone I would like it to be either a new segment or an option to the script. Segment is preferable.

**Comment by manasb** [comment]
@BrianEnigma I tried those changes and they seem to work well. Thank you.

**Comment by LestaD** [comment]
@b-ryan maybe add segment to split line?
And i want to add empty line **before** powerline

**Comment by LestaD** [comment]
@milkbikis can be realized split line?

**Comment by b-ryan** [comment]
Implemented in #266 - let me know if there are any issues
